### DISCUSSION TITLE
How to show program paths and files paths in search result ?

### DISCUSSION THREAD

**Discussion by pchris7489** [Open-Shell/Open-Shell-Menu#1614] [General]
Hello,

I find the search results sometimes confusing. There are indeed categories when you type a search, it will show either the result is a program, a file, a control panel setting.

But more often than not, there are multiple same names in the result sub-menu, making it hard to know what to open. So usually, the mouse cursor must be placed on the item to have the details.

For example, search for "Notepad". It will bring the notepad shortcut and the notepad executable. If I want to show the notepad executable properties and not the shortcut, I can't know what to select in the first place.

How can I mod the OpenShell configuration to display the file/program path and/or the date of modification ? I think that OpenShell is using Windows Search API where those informations are already available

Thank you

**Comment by ge0rdi** [comment]
> How can I mod the OpenShell configuration to display the file/program path and/or the date of modification ?

I think that's currently not possible.
We have those paths available, so I guess it should be possible to add option to display paths (in tooltip or maybe in search results window).
You can create _feature request_ for that.
Though note that there is not much development going on here, so don't expect that it will be done soon (if at all).

We can also improve deduplication of search results (we are doing some deduplication already).
There is not much sense to display Notepad shortcut and notepad.exe at the same time.
I guess that would help with your issue too.

**Comment by pchris7489** [comment]
actually it is very useful to display both the notepad shortcut and the notepad executable at the same time, I chose this example because it serves as a common reference for everyone, however, for more personalized software like image editors, word files, or even software source files like javascript or c++ files, or when packaging software, modifying the shortcut to change how the file opens or selecting the destination file in order to change or move a component in the source destination, having both shown in the search results is highly beneficial

the real issue lies in quickly distinguishing between the two. currently, I find myself manually hovering the mouse over the item or performing multiple clicks to access the file's properties, which is counterintuitive because the search feature is supposed to enhance efficiency

thank you for your feedback
